Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using DTO;
+using Kernel;
+using System;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class CreateUserCommand : ICreateUserCommand
```

Comment:
may be rename interface ICreateUserCommand and IDeleUserComand to Comand? They contains identic method Execute

Reply:
They will have similar Input/output signature. So, it is only the first method will be implemented always